### DISCUSSION TITLE
Is there a way to connect with portainer?

### DISCUSSION THREAD

**Discussion by JimmyKmi** [logto-io/logto#5227] [Q&A] [answered]
I just try to use portainer's Authentication via Logto's SSO, and It shows "resource indicator must be an absolute URI"

I created an Traditional Web app in logto, and here's my config in portainer:
![screenshot-20240112-133636](https://github.com/logto-io/logto/assets/69306141/b5e5679f-1cc4-40c5-a28e-2f02aed48d8d)

**Comment by charIeszhao** [comment]
It seems OK from your screenshot. The resource indicator should be `http://logto-url/oidc/token` in your case. Where did the error come from? Is it from Logto console or from the OIDC service during navigation?

**Reply by JimmyKmi** [reply]
It is from OIDC service during navigation.

After I click "Login with OAuth" in

https://[portainer URL]/

the url change like this:

https://<portainer URL>/?error=invalid_target&error_description=resource%20indicator%20must%20be%20an%20absolute%20URI&state=bf9d7715-549b-4741-ad5d-bd6155a26621&iss=[portainer URL]%2Foidc#!/auth

**Reply by wangsijie** [reply]
Are you using Logto as third party app?

**Reply by JimmyKmi** [reply]
I'm using Logto as a central authentication service

**Reply by wangsijie** [reply]
Everything looks good in your screenshot, it should ok. There is one thing to check: are you using Logto Cloud or OSS? Try to change the endpoints to "https". We have a similar example with OAuth, see if this blog helps: https://blog.logto.io/integrate-with-passportjs/

**Reply by JimmyKmi** [reply]
I‘m using Logto Self-host for test, be like: Docker(http) -> CDN(with https) -> client(https)
I tried change the endpoints to "https" but there is no change.

**Reply by wangsijie** [reply, marked as answer]
The issue is very likely on the Portainer side. If there are still problems, it is recommended to provide more detailed information.
